docs/production-deployment/cloud/gcp-export-gcs.mdx

```diff
@@ -0,0 +1,166 @@
+---
+id: gcp-export-gcs
+title: Exporting Workflow Event History to GCS
+sidebar_label: GCP Export to GCS
+description: Export Workflow History to GCS
+slug: /cloud/export/gcp-export-gcs
+toc_max_heading_level: 4
+keywords:
+  - explanation
+  - how-to
+  - operations
+  - temporal cloud
+  - term
+  - verify
+  - workflow history export
+  - gcp
+tags:
+  - explanation
+  - how-to
+  - operations
+  - temporal-cloud
+  - term
+  - verify
+  - workflow-history-export
+  - gcp
+---
+
+## **Prerequisites**
```

Comment:
I would unbold these? Unless it's for a particular reason but it doesn't match the rest of the docs.

Reply:
Fixed